cmd/gen-go-sample/inittree.go

```diff
@@ -511,3 +515,21 @@ func snakeToCamel(s string) string {
 	}
 	return sb.String()
 }
+
+func printCommentLines(w *bufio.Writer, comment string, ind int) {
+	if comment == "" {
+		return
+	}
+	for _, c := range strings.Split(strings.TrimRight(comment, "\n"), "\n") {
+		indent(w, ind)
+		w.WriteString("// ")
+		w.WriteString(c)
+		w.WriteByte('\n')
+	}
+}
+
+func indent(w *bufio.Writer, ind int) {
```

Comment:
I would suggest `indent` return a `string`, and  `printCommentLines` could simply use that

Reply:
`indent` is also used elsewhere. (It's also not new code either).